src/Microphysics2M.jl

```diff
@@ -447,6 +447,103 @@ function rain_evaporation(
     return (evap_rate_0, evap_rate_1)
 end
 
+""" 
+    radar_reflectivity(struct, struct, q_liq, q_rai, N_liq, N_rai, ρ_air, ρ_w)
+
+    - `struct` - type for 2-moment rain autoconversion parameterization
+    - `νr` - rain gamma distribution mass coefficient
+    - `μr` -  rain gamma distribution exponential coefficient 
+    - `q_rai` - rain water specific humidity
+    - `N_liq` - cloud droplet number density 
+    - `N_rai` - rain droplet number density 
+    - `ρ_air` - air density
+    - `ρ_w` - water density
+
+Returns logarithmic radar reflectivity from the assumed cloud and rain particle 
+size distribuions normalized by the reflectivty of 1 millimiter drop in a volume of
+one meter cube
+"""
+function radar_reflectivity(
+    acnv::CMP.AcnvSB2006{FT},
+    νr::FT,
```

Comment:
Lets not pass them in as arguments. We can just assume exponential distribution for ran